skimage/restoration/non_local_means.py

```diff
@@ -14,14 +14,14 @@
 def denoise_nl_means(image, patch_size=7, patch_distance=11, h=0.1,
                      multichannel=False, fast_mode=True, sigma=0., *,
                      preserve_range=False, channel_axis=None):
-    """Perform non-local means denoising on 2-D or 3-D grayscale images, and
-    2-D RGB images.
+    """Perform non-local means denoising on 2D-4D grayscale or RGB images.
```

Comment:
While shortening to one line, I fixed an outdated description of the supported dimensions.